### ISSUE TITLE
Add switch into ARM template to allow user to pick between deploying Mongo on IAAS or provide an already existing Mongo connection string

### ISSUE THREAD

**Issue by martinpeck** [opencb/opencga#1036]

**Comment by martinpeck** [comment]
@lawrencegripper can you please make sure the title and description reflect the task, and then we can assign elsewhere? Thanks!

**Comment by lawrencegripper** [comment]
@martinpeck I've updated the title and added some more info in the description. I think the outstanding questions for this issue are:

- Is this something that is needed for the solution?
- If it is needed at what point do we do this work?

IMO it's a nice-to-have which could be completed after the perf testing phase.

**Comment by martinpeck** [comment]
I agree. Unless we find that....

- we're totally unable to tweak a perf related issue
- we're sure that MongoDB is the bottle next for this perf issue
- we're sure that using a remote MongoDB, outside of Azure, would fix it

... that we can push this out of scope for the perf testing.

**Comment by lawrencegripper** [comment]
Makes sense to me, should we have a label for `low-priority` or something similar and tag things like this?
